hypothesis-python/tests/cover/test_lookup.py

```diff
@@ -119,17 +124,17 @@ def test_specialised_collection_types(data, typ, coll_type, instance_of):
     assume(instances)  # non-empty collections without calling len(iterator)
 
 
-@given(from_type(typing.DefaultDict[int, int]).filter(len))
+@given(from_type(typing.DefaultDict[Elem, Elem]).filter(len))
```

Comment:
I guess, in the future, it would be better to support two different types for keys and values.

This way we can be sure that the generated value holds the described structure better.